tests/test_gui_api.py

```
@@ -0,0 +1,97 @@
+    assert "does not exist" in response["error"]
+
+
+def test_plan_job_uses_service_and_serializes_report(monkeypatch, tmp_path):
+    video = tmp_path / "Example.mkv"
+    output = tmp_path / "Example_Plex.mkv"
+    plan = MuxPlan(source_video=video, output_path=output)
+
+    monkeypatch.setattr("plexmuxy_gui.api.load_or_default_config", default_config)
+    monkeypatch.setattr(
+        "plexmuxy_gui.api.run_mux_job",
+        lambda input_dir, config, dry_run, yes: JobReport(input_dir=input_dir, plans=[plan]),
```

Ruff (0.15.20)

[warning] Unused lambda argument: `config`

(ARG005)